website/docs/d/oracle_autonomous_database_regular.html.markdown

```diff
@@ -0,0 +1,156 @@
+
+* `cpu_core_count` - The number of CPU cores to be made available to the database. When the ECPU is selected, the value for cpuCoreCount is 0. For Autonomous Database on Dedicated Exadata infrastructure, the maximum number of cores is determined by the infrastructure shape. See [Characteristics of Infrastructure Shapes](https://www.oracle.com/pls/topic/lookup?ctx=en/cloud/paas/autonomous-database&id=ATPFG-GUID-B0F033C1-CC5A-42F0-B2E7-3CECFEDA1FD1) for shape details.
+
+* `data_storage_size_in_gbs` - The quantity of data in the database, in gigabytes.
+
+* `data_storage_size_in_tbs` - The quantity of data in the database, in terabytes.
+
+* `db_node_storage_size_in_gbs` - The DB node storage size in, in gigabytes.
+
+* `db_version` - A valid Oracle Database version for Autonomous Database.
+
+* `display_name` - The user-friendly name for the Autonomous Database. The name does not have to be unique.
+
+* `failed_data_recovery_in_seconds` - Indicates the number of seconds of data loss for a Data Guard failover.
+
+* `in_memory_area_in_gbs` - The area assigned to In-Memory tables in Autonomous Database.
+
+* `auto_scaling_enabled` - Indicates if auto scaling is enabled for the Autonomous Database CPU core count.
+
+* `auto_scaling_for_storage_enabled` - Indicates if auto scaling is enabled for the Autonomous Database storage.
+
+* `local_data_guard_enabled` - Indicates whether the Autonomous Database has local (in-region) Data Guard enabled. Not applicable to cross-region Autonomous Data Guard associations, or to Autonomous Databases using dedicated Exadata infrastructure or Exadata Cloud@Customer infrastructure.
+
+* `mtls_connection_required` - Specifies if the Autonomous Database requires mTLS connections.
+
+* `preview` - Indicates if the Autonomous Database version is a preview version.
+
+* `preview_version_with_service_terms_accepted` - Indicates if the Autonomous Database version is a preview version with service terms accepted.
+
+* `remote_data_guard_enabled` - Indicates whether the Autonomous Database has Cross Region Data Guard enabled. Not applicable to Autonomous Databases using dedicated Exadata infrastructure or Exadata Cloud@Customer infrastructure.
+
+* `key_history_entry` - Key History Entry.
+
+* `lifecycle_details` - Information about the current lifecycle state.
+
+* `local_adg_auto_failover_max_data_loss_limit` - Parameter that allows users to select an acceptable maximum data loss limit in seconds, up to which Automatic Failover will be triggered when necessary for a Local Autonomous Data Guard
+
+* `location` - The Azure Region where the Autonomous Databases exists.
+
+* `memory_per_oracle_compute_unit_in_gbs` - The amount of memory (in GBs) enabled per ECPU or OCPU.
+
+* `ncharacter_set` - The national character set for the autonomous database.  The default is AL16UTF16. Allowed values are: AL16UTF16 or UTF8.
```

Comment:
please quote all values
```suggestion
* `ncharacter_set` - The national character set for the autonomous database.  The default is `AL16UTF16`. Allowed values are: `AL16UTF16` or `UTF8`.
```

Reply:
ncharacter_set is ver ambigious, i would like this to be 
```suggestion
* `national_character_set` - The national character set for the autonomous database.  The default is AL16UTF16. Allowed values are: AL16UTF16 or UTF8.
```

Reply:
Done.